hail/python/hailtop/aiocloud/aioazure/fs.py

```diff
@@ -368,6 +368,10 @@ def __init__(self, *, credential_file: Optional[str] = None, credentials: Option
         self._credential = credentials.credential
         self._blob_service_clients: Dict[Tuple[str, str, Union[AzureCredentials, str, None]], BlobServiceClient] = {}
 
+    @staticmethod
+    def schemes() -> Set[str]:
+        return {'hail-az', 'https'}
```

Comment:
Side-note: would you mind making an issue to drop support for `hail-az`? I believe we just need to update our terraform to use `https` in the global config and then remove it from the code base.

Reply:
https://github.com/hail-is/hail/issues/14116